src/src/DataStructs/EthernetEventData.h

```diff
@@ -43,6 +43,9 @@ struct EthernetEventData_t {
 
 
   void markGotIP();
+#if FEATURE_USE_IPV6
+  void martGotIP6(const IPAddress& ip6);
```

Comment:
I'd expected that toe be named `markGotIPv6`, perhaps? (so, with a `k` instead of a `t`, and maybe a `v` inserted ?)

Reply:
Nah, Mart got the 6th IP, that's all... ;)